labs/RivePlayer/samples/RivePlayer.Samples/RivePlayerCustomSample.xaml.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommunityToolkit.Labs.WinUI.Rive;
+
+namespace RivePlayerExperiment.Samples;
+
+/// <summary>
+/// An example sample page of a custom control inheriting from Panel.
+/// </summary>
+[ToolkitSample(id: nameof(RivePlayerCustomSample), "Custom control", description: $"A sample for showing how to create and use a {nameof(RivePlayer)} custom control.")]
+public sealed partial class RivePlayerCustomSample : Page
+{
+    public RivePlayerCustomSample()
+    {
+        this.InitializeComponent();
+#if HAS_UNO
+        // The Uno WASM platform seems to have trouble with our "ms-appx://" URI. Give it an http.
+        this.RivePlayer.Source = "https://public.rive.app/community/runtime-files/2244-4463-animated-login-screen.riv";
```

Comment:
This is generally related to the way the `web.config` file is setup (IIS does not serve unknown mime types by default), which prevents `.riv` files to be downloaded properly.

Reply:
Thanks for explaining that. This makes sense. I added ".riv" files to web.config and the JS console error went away, but the animation still doesn't load. Anything else I'm missing? Navigating to "https://localhost:5001/animated-login-screen.riv" just loads the same XAML page again (doesn't download  .riv file).

Reply:
This still means that the extension is filtered somehow `.riv`. Changing the `web.config` should have been enough. Does it work if you call it `.dat` ?

Reply:
Calling it `.dat` does not work and doesn't print an error to the web developer console either.

Should I be able to navigate to this file somewhere when the local server is running? "https://localhost:5001/animated-login-screen.dat" maybe?